--- origsrc/speech-dispatcher-0.6.7/src/audio/Makefile.am	2007-12-17 02:53:09.000000000 -0600
+++ src/speech-dispatcher-0.6.7/src/audio/Makefile.am	2010-05-31 22:04:54.216749800 -0500
@@ -23,7 +23,8 @@ endif
 EXTRA_DIST = alsa.c oss.c nas.c pulse.c
 
 AM_CFLAGS = $(am_cflags) $(NAS_FLAGS) $(PULSE_FLAGS) $(ALSA_FLAGS)
-libsdaudio_la_LDFLAGS = -version-info @LIB_SDAUDIO_CURRENT@:@LIB_SDAUDIO_REVISION@:@LIB_SDAUDIO_AGE@ -lpthread $(NAS_LIBS) $(PULSE_LIBS) $(ALSA_LIBS)
+libsdaudio_la_LDFLAGS = -version-info @LIB_SDAUDIO_CURRENT@:@LIB_SDAUDIO_REVISION@:@LIB_SDAUDIO_AGE@ -no-undefined
+libsdaudio_la_LIBADD = -lpthread $(NAS_LIBS) $(PULSE_LIBS) $(ALSA_LIBS)
 
 spdlib_LTLIBRARIES = libsdaudio.la
 
--- origsrc/speech-dispatcher-0.6.7/src/c/api/Makefile.am	2008-07-30 04:51:20.000000000 -0500
+++ src/speech-dispatcher-0.6.7/src/c/api/Makefile.am	2010-05-31 22:05:22.977394800 -0500
@@ -8,6 +8,6 @@ lib_LTLIBRARIES = libspeechd.la
 libspeechd_la_SOURCES = libspeechd.c
 libspeechd_la_HEADERS = libspeechd.h
 libspeechd_ladir = $(includedir)
-libspeechd_la_LDFLAGS = -version-info @LIB_SPD_CURRENT@:@LIB_SPD_REVISION@:@LIB_SPD_AGE@ -lpthread
+libspeechd_la_LDFLAGS = -version-info @LIB_SPD_CURRENT@:@LIB_SPD_REVISION@:@LIB_SPD_AGE@ -no-undefined -lpthread
 libspeechd_la_LIBADD = @glib_libs@
 
