--- origsrc/speech-dispatcher-0.6.7/config/clients/Makefile.am	2008-07-09 02:55:25.000000000 -0500
+++ src/speech-dispatcher-0.6.7/config/clients/Makefile.am	2010-05-31 23:10:26.223647700 -0500
@@ -1,5 +1,5 @@
 ## Process this file with automake to produce Makefile.in
 
-dist_clientconf_DATA = *.conf
-dist_clientconforig_DATA = *.conf
+dist_clientconf_DATA = $(wildcard $(srcdir)/*.conf)
+dist_clientconforig_DATA = $(wildcard $(srcdir)/*.conf)
 
--- origsrc/speech-dispatcher-0.6.7/config/Makefile.am	2008-07-09 02:55:21.000000000 -0500
+++ src/speech-dispatcher-0.6.7/config/Makefile.am	2010-05-31 23:11:08.161046400 -0500
@@ -5,7 +5,7 @@ SUBDIRS= modules clients
 all-local: speechd.conf
 
 speechd.conf: speechd.conf.in
-	sed -e s:INCLUDE_PATH:\"$(spdconfdir)\":g speechd.conf.in > speechd.conf
+	sed -e s:INCLUDE_PATH:\"$(spdconfdir)\":g $< > $@
 
 
 spdconf_DATA= speechd.conf
--- origsrc/speech-dispatcher-0.6.7/config/modules/Makefile.am	2008-07-09 02:55:29.000000000 -0500
+++ src/speech-dispatcher-0.6.7/config/modules/Makefile.am	2010-05-31 23:09:26.158212100 -0500
@@ -1,5 +1,5 @@
 ## Process this file with automake to produce Makefile.in
 
-dist_moduleconf_DATA = *.conf
-dist_moduleconforig_DATA = *.conf
+dist_moduleconf_DATA = $(wildcard $(srcdir)/*.conf)
+dist_moduleconforig_DATA = $(wildcard $(srcdir)/*.conf)
 
--- origsrc/speech-dispatcher-0.6.7/configure.in	2008-07-30 08:27:54.000000000 -0500
+++ src/speech-dispatcher-0.6.7/configure.in	2010-05-31 22:08:10.192959000 -0500
@@ -8,7 +8,7 @@ dnl progs
 AC_PROG_CC
 
 dnl info
-AM_INIT_AUTOMAKE("speech-dispatcher", "`cat version.sh|grep '^VERSION'|awk -F= '{print $2}'`")
+AM_INIT_AUTOMAKE("speech-dispatcher", "`cat $srcdir/version.sh|grep '^VERSION'|awk -F= '{print $2}'`")
 
 AM_CONFIG_HEADER(config.h)
 
--- origsrc/speech-dispatcher-0.6.7/src/c/clients/say/Makefile.am	2004-05-23 16:23:33.000000000 -0500
+++ src/speech-dispatcher-0.6.7/src/c/clients/say/Makefile.am	2010-05-31 22:59:40.273701500 -0500
@@ -7,4 +7,4 @@ AM_CFLAGS = -DLOCALEDIR=\"$(localedir)\"
 
 bin_PROGRAMS = spd-say
 spd_say_SOURCES = say.c options.c options.h $(c_api)/libspeechd.h
-spd_say_LDADD = $(c_api)/libspeechd.la
+spd_say_LDADD = $(top_builddir)/src/c/api/libspeechd.la
--- origsrc/speech-dispatcher-0.6.7/src/modules/Makefile.am	2008-06-27 07:30:01.000000000 -0500
+++ src/speech-dispatcher-0.6.7/src/modules/Makefile.am	2010-05-31 23:01:06.660642500 -0500
@@ -7,7 +7,7 @@ SNDFILE_LIBS = @SNDFILE_LIBS@
 
 EXTRA_DIST = module_main.c module_utils_addvoice.c festival_client.c festival_client.h ivona_client.c dummy.c dummy-message.wav
 
-AM_CFLAGS = -Wall -DLOCALEDIR=\"$(localedir)\" -DDATADIR=\"$(snddatadir)\" -I/usr/include/ $(inc_local) @glib_include@ -L$(top_srcdir)/src/audio -I$(top_srcdir)/src/audio -I../../intl/ $(ibmtts_include) @SNDFILE_CFLAGS@ -Wl,--rpath -Wl,$(spdlibdir)
+AM_CFLAGS = -Wall -DLOCALEDIR=\"$(localedir)\" -DDATADIR=\"$(snddatadir)\" -I/usr/include/ $(inc_local) @glib_include@ -L$(top_builddir)/src/audio -I$(top_srcdir)/src/audio -I../../intl/ $(ibmtts_include) @SNDFILE_CFLAGS@ -Wl,--rpath -Wl,$(spdlibdir)
 
 modulebin_PROGRAMS = sd_dummy sd_generic sd_festival sd_cicero
 
--- origsrc/speech-dispatcher-0.6.7/src/server/config.c	2008-06-09 05:46:34.000000000 -0500
+++ src/speech-dispatcher-0.6.7/src/server/config.c	2010-06-01 01:48:52.088352100 -0500
@@ -26,7 +26,7 @@
 
 #include "speechd.h"
 #include "config.h"
-#include "intl/fdsetconv.h"
+#include "fdsetconv.h"
 
 static TFDSetClientSpecific *cl_spec_section;
 
--- origsrc/speech-dispatcher-0.6.7/src/tests/Makefile.am	2008-04-09 06:39:39.000000000 -0500
+++ src/speech-dispatcher-0.6.7/src/tests/Makefile.am	2010-05-31 23:02:56.296913400 -0500
@@ -8,18 +8,18 @@ AM_CFLAGS = -I$(c_api) -DLOCALEDIR=\"$(l
 bin_PROGRAMS = long_message clibrary clibrary2 run_test connection_recovery
 
 long_message_SOURCES = long_message.c $(c_api)/libspeechd.h
-long_message_LDADD = $(c_api)/libspeechd.la
+long_message_LDADD = $(top_builddir)/src/c/api/libspeechd.la
 
 clibrary_SOURCES = clibrary.c $(c_api)/libspeechd.h
-clibrary_LDADD = $(c_api)/libspeechd.la
+clibrary_LDADD = $(top_builddir)/src/c/api/libspeechd.la
 
 clibrary2_SOURCES = clibrary2.c $(c_api)/libspeechd.h
-clibrary2_LDADD = $(c_api)/libspeechd.la
+clibrary2_LDADD = $(top_builddir)/src/c/api/libspeechd.la
 
 connection_recovery_SOURCES = connection-recovery.c $(c_api)/libspeechd.h
-connection_recovery_LDADD = $(c_api)/libspeechd.la
+connection_recovery_LDADD = $(top_builddir)/src/c/api/libspeechd.la
 
 run_test_SOURCES = run_test.c $(c_api)/libspeechd.h
-run_test_LDADD = $(c_api)/libspeechd.la
+run_test_LDADD = $(top_builddir)/src/c/api/libspeechd.la
 
 EXTRA_DIST= *.test *.wav
